module/move/willbe/Cargo.toml

```diff
@@ -62,6 +62,7 @@ handlebars = "4.5.0"
 ureq = "~2.9"
 colored = "2.1.0"
 duct = "0.13.7"
+uuid = {  version = "1.7.0", features = [ "v4" ] }
```

Comment:
Do we really need this dependency?

Reply:
Can you please suggest how I can solve the problem of generating a unique name for a temporary folder?

Reply:
Sure, an alternative way to generate unique folder names could be by incorporating a timestamp and check if a folder already exists before creating it.